content/en/docs/examples/virtual-machines/bookinfo/index.md

```diff
@@ -19,20 +24,20 @@ VMs cannot initiate any direct communication to Kubernetes Pods even when using
 
 ## Overview
 
-{{< image width="80%" link="./mesh-expansion.svg" caption="Bookinfo Application with Istio Mesh Expansion" >}}
+{{< image width="80%" link="./vm-bookinfo.svg" caption="Bookinfo running on VMs" >}}
 
 <!-- source of the drawing
 https://docs.google.com/drawings/d/1G1592HlOVgtbsIqxJnmMzvy6ejIdhajCosxF1LbvspI/edit
  -->
 
 ## Before you begin
 
-* Setup Istio by following the instructions in the
+- Setup Istio by following the instructions in the
   [Installation guide](/docs/setup/install/kubernetes/).
 
-* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application (in the `bookinfo` namespace).
+- Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application (in the `bookinfo` namespace).
 
-* Create a VM named 'vm-1' in the same project as Istio cluster, and [Join the Mesh](/docs/examples/mesh-expansion/single-network/).
+- Create a VM named 'vm-1' in the same project as Istio cluster, and [Join the Mesh](/docs/examples/virtual-machines/single-network/).
```

Comment:
```suggestion
- Create a VM named 'vm-1' in the same project as the Istio cluster, and [join the Mesh](/docs/examples/virtual-machines/single-network/).
```

Reply:
Is the use of "project" correct here? Thats a GCP-ism.

Reply:
This whole example is Google-specific. Rigs and I agreed to just fix the terminology for now, but after 1.4 somebody should rework the example to work on any platform.